report/src/main/webapp/app/layout/sidepanel/sidepanel.view.html

```diff
@@ -116,7 +116,9 @@
             data-trigger="hover" data-container="body">{{sidepanel.testsStats.warning}}</span> /
       <span class="status passed" data-toggle="popover" data-content="Passed tests"
             data-trigger="hover" data-container="body">
-        {{sidepanel.testsStats.passed + sidepanel.testsStats.conditionallyPassed}}
+        {{sidepanel.testsStats.passed}} </span>
```

Comment:
Why did you remove this addition here? I think that by design the number of displayed Passed tests should also include the number of conditional passed ("conditional pass" is a special case of "pass")

Reply:
Sorry for that! Fixed!